contrib/inventory/proxmox.py

```diff
@@ -212,13 +214,17 @@ def main_host(options):
     return {}
 
 def main():
+
+    bool_validate_cert = False if os.environ.has_key('PROXMOX_INVALID_CERT') else True
+
     parser = OptionParser(usage='%prog [options] --list | --host HOSTNAME')
     parser.add_option('--list', action="store_true", default=False, dest="list")
     parser.add_option('--host', dest="host")
     parser.add_option('--url', default=os.environ.get('PROXMOX_URL'), dest='url')
     parser.add_option('--username', default=os.environ.get('PROXMOX_USERNAME'), dest='username')
     parser.add_option('--password', default=os.environ.get('PROXMOX_PASSWORD'), dest='password')
     parser.add_option('--pretty', action="store_true", default=False, dest='pretty')
+    parser.add_option('--trust-invalid-certs', action="store_true", default=bool_validate_cert, dest='validate')
```

Comment:
So, if I am not mistaken, if I do not use the option, valide is True. If I use the option, then store_true will mean that validate will also be True. Shouldn't it be the reverse ?